code/controllers/subsystem/jobs.dm

```diff
@@ -555,7 +568,7 @@ SUBSYSTEM_DEF(jobs)
 			if(job.get_exp_restrictions(player.client))
 				young++
 				continue
-			if(job.barred_by_disability(player.client))
+			if(job.barred_by_disability(player.client) || job.barred_by_missing_limbs(player.client))
```

Comment:
Might be nice to gather data separately for this instead of putting it with the disability data.

Reply:
I'm usually on the side of collecting data, but in this case, I don't think separate data would be useful. Both a genetic and a limb disability is the same in terms of what we are looking here - someone tried to make a character that was physically unfit for a restricted job!